### ISSUE TITLE
Delete audio after transcription

### ISSUE THREAD

**Issue by bepolymathe** [Synaptic-Labs-AI/NeuroVox#4]
Hi, 

For the short notes of everyday life, it is not always necessary to save the audio file so as not to make the trunk size excessively magnify. 
Would it be possible to make such an option?

**Comment by ProfSynapse** [comment]
What I can do is have some modal popup no matter what after you hit stop to ask if you want to save the actual recording. 

Will plan for next update already have that functionality when you X out of the recording modal will just make it global.

**Comment by ChrisBBBB** [comment]
I don't want to appear unhelpful, but the option of saving or not saving in the modal box wouldn't work for some people because, for example, I like to see that the transcription has actually worked properly, so I want to keep the recording until the whole process is successfully completed. If you can make that part of the process, then great, but I wouldn't want a recording to be deleted if then the AI model fails and doesn't provide the transcription.

**Comment by ProfSynapse** [comment]
yeah going to remove this options people can always delete the audio
